model_data_preprocess/postprocessing_E3SM_data_for_TC_analysis.sh

```diff
@@ -1,100 +1,84 @@
 #!/bin/bash
 
 # A Bash script to post-process E3SM 6 hourly (h2) instantaneous output to generate a text file storing Tropical Cyclone tracks
-# tempestremap and tempestextremes are built in e3sm-unified from version 1.5.0. This script has been tested using Cori login node.
-# To request an interactive session as follows: salloc --nodes=1 --partition=debug --time=30:00 -C haswell
+# tempestremap and tempestextremes are built in e3sm-unified from version 1.5.0. This script has been tested using Perlmutter login node.
+# To request an interactive session as follows: salloc --nodes=1 --partition=debug --time=30:00 -C cpu
 
-source /global/common/software/e3sm/anaconda_envs/load_latest_e3sm_unified_cori-haswell.sh
+source /global/common/software/e3sm/anaconda_envs/load_latest_e3sm_unified_pm-cpu.sh
 
 #For typical EAM v2 ne gp2 grids.
-start="0051"
-end="0060"
-caseid="20210528.v2rc3e.piControl.ne30pg2_EC30to60E2r2.chrysalis"
-drc_in=/global/cfs/cdirs/e3smdata/zppy_complete_run_nersc_output/${caseid}/archive/atm/hist
-# Warning: because somehow tempest-remap only can writes grid file on SCRATCH space. The resulted files will be moved to another path at the end.
-result_dir_fin=/global/cfs/cdirs/e3sm/e3sm_diags/postprocessed_e3sm_v2_data_for_e3sm_diags/${caseid}/
-mkdir -p $result_dir_fin
-result_dir=$SCRATCH/tests/tc-analysis/
```

Comment:
With Perlmutter, we don't need to use `$SCRATCH` anymore? For `zppy`, we were using it for all machines:
```
# Warning: tempest-remap can only write grid file on SCRATCH space.
# The result files will be moved to another path at the end.
result_dir_fin={{ output }}/post/atm # Directory will be {{ output }}/post/atm/tc-analysis_${Y1}_${Y2}
mkdir -p $result_dir_fin
y1={{ year1 }}
y2={{ year2 }}
Y1="{{ '%04d' % (year1) }}"
Y2="{{ '%04d' % (year2) }}"
result_dir={{ scratch }}/tc-analysis_${Y1}_${Y2}/
```

Reply:
Yes, I believe we don't need to use scratch any more, with fixing over-writing mesh file. At least, my tests worked okay.